source/components-and-apis.rst

```diff
@@ -409,3 +421,5 @@ dojot.
 
 .. _GitHub - Kafka WS: https://github.com/dojot/dojot/tree/v0.5.0/subscription-engine/kafka-ws
 .. _API - kafka-ws: https://dojot.github.io/dojot/kafka-ws/apiary_v0.5.0.html
+
+.. _GitHub - V2: https://github.com/dojot/gui-v2
```

Comment:
Change to https://github.com/dojot/gui-v2/tree/v0.5.0

Reply:
Done